
<launch>
    <param name="Camera_image_topic" type="str" value="/usb_cam/image_raw" />
    <param name="fps" type="int" value="30" />
    <param name="width" type="int" value="640" />
    <param name="height" type="int" value="480" />
    <param name="server_ip" type="str" value="10.147.20.197" />
    <node name="node_name" pkg="videoStreamer" type="videoStreamer.py" output="screen"/>
</launch>
